model/url.yaml

```diff
@@ -37,3 +37,94 @@ groups:
         type: string
         brief: 'The [URI fragment](https://www.rfc-editor.org/rfc/rfc3986#section-3.5) component'
         examples: ["SemConv"]
+      - id: registered_domain
+        requirement_level: opt_in
+        type: string
+        brief: >
```

Comment:
You need to use `brief: |` here to be able to achieve multiple lines I think. Right now, the table is broken into multiple lines 
![Uploading image.png…]()

It's also the same in the other attributes

Reply:
Even with that the table was kinda broken. I moved some content into the `note` section and only kept the basic definitions in the `brief`.